PocketKit/Sources/Localization/Strings.swift

```diff
@@ -482,8 +482,8 @@ public enum Localization {
     }
     public enum Premium {
       public enum Settings {
-        /// Contact pocket support
-        public static let contactPocketSupport = Localization.tr("Localizable", "settings.premium.settings.contactPocketSupport", fallback: "Contact pocket support")
+        /// Contact Pocket support
```

Comment:
@Gio2018 should there be a en.plist change here too?

Reply:
I missed committing this file in my other PR, so this should go in as well.

https://pocket.slack.com/archives/C01UH57EJKD/p1683216227274079

Reply:
Not sure why this got into this PR, I'll let @cyndichin to add a separate one for this strings